_config.yml

```diff
@@ -28,7 +28,11 @@ exclude: ["Gemfile",
           "scripts/",
           ".gitignore",
           ".sass-cache/",
-          ".idea/"]
+          ".idea/",
+          "README.md",
```

Comment:
This is already excluded at above.  So no need to repeat here.

Reply:
I will remove this one.